### DISCUSSION TITLE
DPI scaling

### DISCUSSION THREAD

**Discussion by irldoggo** [JaKooLit/NixOS-Hyprland#18] [Q&A] [answered]
Probably the best and most fleshed out set of dotfiles I have come across, thanks for that.
Just wondering what my options are for DPI scaling? I am running on a 4K display and waybar and most of hyprland is much much too small for comfortable reading. Do you have options somewhere to address this?

**Comment by JaKooLit** [comment, marked as answer]
https://github.com/JaKooLit/Hyprland-Dots/wiki/TIPS#%EF%B8%8F-adjusting-fonts-sizes-and-font-scaling-including-waybar-fonts


I dont have 4k but on my 2k monitor, I only adjust the font and font scaling using nwg-look

however, you can also do a combination

first is set a scaling wanted
Super E monitors

on this line for example
monitor=,preferred,auto,1

1 is the scale. You can first try 1.25 or 1.5 or even auto
1 = 100%
so 1.5 is 150% scaling

after which, you can tweak. On the link I placed at the beginning is a tip

**Comment by irldoggo** [comment]
Perfect, it's much better now for my old man eyes. Thanks!

**Reply by JaKooLit** [reply]
great. yw
